### DISCUSSION TITLE
How should trading work in the server?

### DISCUSSION THREAD

**Discussion by ar00n** [CraftRealm/management#38] [General]
Should trading be enabled via AuctionHouse or should it just be player shops and town trade links. Or any other alternatives.

**Comment by AngelInTheHeaven** [comment]
player shops so it would make more reason to make players visit other people's town

**Reply by ar00n** [reply]
I second this. I'm thinking of making spawn just exactly that with no core gameplay features. Other than the ship to randomly teleport. Thinking of adding a trade plugin that can only be used at close distance.
